config/kubernetes-sigs/sig-auth/teams.yaml

```diff
@@ -7,12 +7,16 @@ teams:
     - srampal
     - tashimi
     privacy: closed
+    repos:
+      hierarchical-namespaces: admin
```

Comment:
does this construction mean that any sig-level approver (e.g. https://github.com/kubernetes/org/blob/main/config/kubernetes-sigs/sig-auth/OWNERS) can approve a config change like this to grants admin permissions to any kubernetes-sigs repo? if so, that doesn't seem great.

Reply:
Hi @liggitt!

While a sig-level approver can approve a config change such as this, for the change to _actually_ take affect, the repo for which the `teams.yaml` file is being modified will have to be added to `restrictions.yaml` for runtime checks to pass as demonstrated here: https://github.com/kubernetes/org/pull/2614#issuecomment-813008435. _Only_ GitHub admins can review/approve changes to `restrictions.yaml`, meaning that sig-level approvers function within the scope of their sig (as defined in `restrictions.yaml`) and any additional repos to be added will have to be reviewed and approved by the GitHub admins, while allowing `peribolos` to safely manage repo permissions.